server/utils/placementCriteriaUtils.ts

```diff
@@ -73,30 +75,25 @@ export const specialistApTypeCriteriaLabels = filterByType<SpecialistApTypeCrite
   specialistApTypeCriteria,
   placementCriteriaLabels,
 )
+
 export const apTypeCriteriaLabels: Record<ApTypeCriteria, string> = {
   normal: 'Standard AP',
   ...specialistApTypeCriteriaLabels,
 }
 
-export const spaceSearchCriteriaApLevelLabels = {
-  acceptsSexOffenders: 'Sexual offences against adults',
-  acceptsChildSexOffenders: 'Sexual offences against children',
-  acceptsNonSexualChildOffenders: 'Non sexual offences against children',
-  isSuitableForVulnerable: 'Vulnerable to exploitation (removes ESAP APs)',
-  isCatered: 'Catered',
-}
-
-export const spaceSearchCriteriaRoomLevelLabels = {
-  isWheelchairDesignated: 'Wheelchair',
-  isStepFreeDesignated: 'Step-free',
-  hasEnSuite: 'En-suite',
-  isSingle: 'Single room',
-  isArsonSuitable: 'Arson room',
-  isSuitedForSexOffenders: 'Suitable for sexual offences',
-}
-
-export const spaceSearchResultsCharacteristicsLabels = {
-  ...spaceSearchCriteriaApLevelLabels,
-  ...spaceSearchCriteriaRoomLevelLabels,
-  isSuitableForVulnerable: placementCriteriaLabels.isSuitableForVulnerable,
+export const apType = (type: ApTypeCriteria): ApType => {
+  switch (type) {
+    case 'isPIPE':
+      return 'pipe'
+    case 'isESAP':
+      return 'esap'
+    case 'isRecoveryFocussed':
+      return 'rfap'
+    case 'isMHAPElliottHouse':
+      return 'mhapElliottHouse'
+    case 'isMHAPStJosephs':
+      return 'mhapStJosephs'
+    default:
+      return 'normal'
+  }
```

Comment:
I think you could do this with a mapping object look-up.

Reply:
Definitely, though I'm tempted to leave it until we have more clarity on how the AP type will be handled with respects to switching to checkboxes -- if this happens!